src/styles/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  margin: 32px auto 40px;
+}
+
+.card__item-name {
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0 16px 4px;
+  color: #060b35;
+}
+
+.card__product-code {
+  font-size: 10px;
+  margin: 0 auto 16px;
+  font-weight: 400;
+  color: #616070;
+  line-height: 14px;
+  margin-left: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
```

Comment:
Same here, let's use general paddings for the card instead of such paddings for the elements